src/main/java/net/mcreator/ui/dialogs/preferences/PreferencesDialog.java

```diff
@@ -173,18 +177,24 @@ private void loadSections() {
 		new PluginsPanel(this);
 
 		themes = new ThemesPanel(this);
-		new EditTemplatesPanel(this, L10N.t("dialog.preferences.page_ui_backgrounds"), "backgrounds", "png");
-
-		new EditTemplatesPanel(this, L10N.t("dialog.preferences.page_procedure_templates"), "templates/ptpl", "ptpl");
-		new EditTemplatesPanel(this, L10N.t("dialog.preferences.page_feature_templates"), "templates/ftpl", "ftpl");
-		new EditTemplatesPanel(this, L10N.t("dialog.preferences.page_ai_builder_templates"), "templates/aitpl",
-				"aitpl");
-		new EditTemplatesPanel(this, L10N.t("dialog.preferences.page_cmd_builder_templates"), "templates/cmdtpl",
-				"cmdtpl");
-		new EditTemplatesPanel(this, L10N.t("dialog.preferences.page_texture_templates"),
-				"templates/textures/texturemaker", "png");
-		new EditTemplatesPanel(this, L10N.t("dialog.preferences.page_armor_templates"), "templates/textures/armormaker",
-				"png");
+
+		addEditTemplatesPanel("ui_backgrounds", "backgrounds", "png");

```

Comment:
The background is not in the templates folder, thus rendering the background useless and not working. Please, go through your changes when you do them and test them.

I can imagine seeing how changing this will work, but one should never assume and test all affected components.

Reply:
Also test if other templates still work

Reply:
I tested the 2 others, but not this one 🤦😓